docs/site/Order-filter.md

```diff
@@ -67,7 +67,7 @@ Return the three most expensive items, sorted by the `price` property:
 
 ```ts
 await itemRepository.find({
-  order: 'price DESC',
+  order: ['price DESC'],
```

Comment:
@zeekun, it seems like either way is fine. If there are more than 1 properties, then we need to use array. See https://github.com/loopbackio/loopback-next/blob/master/docs/site/Order-filter.md?plain=1#L19-L27. 
Or are you experiencing problems without the array?